portfolio/src/main/webapp/hobbies.html

```diff
@@ -112,9 +114,16 @@ <h3 id="xstitch">Cross Stitch</h3>
         There are some pictures of my own cross stitches in the gallery,
         but here are some images of patterns I am currently working on,
         which have been completed by other people.
+    </p>
     <h5>Pretty Little Tokyo</h5>
+    <img src="/images/prettylittletokyo.jpg" alt="A cross stitch of a cute,
+        cartoon-ish rendition of Tokyo." class="patterns"/>
     <h5>Japanese Garden</h5>
+    <img src="/images/japanesegarden.jpg" alt="An elaborate cross stitch of
```

Comment:
These are so impressive!

Reply:
I didn't make this one unfortunately. It's a sample picture! But I'm working on it...